github/dependency_graph.go

```diff
@@ -28,12 +28,32 @@ type CreationInfo struct {
 type RepoDependencies struct {
 	SPDXID *string `json:"SPDXID,omitempty"`
 	// Package name
-	Name             *string `json:"name,omitempty"`
-	VersionInfo      *string `json:"versionInfo,omitempty"`
-	DownloadLocation *string `json:"downloadLocation,omitempty"`
-	FilesAnalyzed    *bool   `json:"filesAnalyzed,omitempty"`
-	LicenseConcluded *string `json:"licenseConcluded,omitempty"`
-	LicenseDeclared  *string `json:"licenseDeclared,omitempty"`
+	Name             *string        `json:"name,omitempty"`
+	VersionInfo      *string        `json:"versionInfo,omitempty"`
+	DownloadLocation *string        `json:"downloadLocation,omitempty"`
+	FilesAnalyzed    *bool          `json:"filesAnalyzed,omitempty"`
+	LicenseConcluded *string        `json:"licenseConcluded,omitempty"`
+	LicenseDeclared  *string        `json:"licenseDeclared,omitempty"`
+	ExternalRefs     []*ExternalRef `json:"externalRefs"`
+}
+
+// ExternalRef represents an external reference (e.g., PURL/SWID/CPE) for a package in the SBOM.
+type ExternalRef struct {
+	ReferenceCategory string `json:"referenceCategory"`
+	ReferenceType     string `json:"referenceType"`
+	ReferenceLocator  string `json:"referenceLocator"`
+}
+
+// Relationship represents a relationship between two packages in the SBOM.
+type Relationship struct {
+	// Element ID
+	SpdxElementID *string `json:"spdxElementId,omitempty"`
```

Comment:
Since SPDX is an acronym for "Software Package Data Exchange", let's please capitalize it following idiomatic Go style:

```suggestion
	SPDXElementID *string `json:"spdxElementId,omitempty"`
```